lua/nvim-tree/lib.lua

```diff
@@ -50,7 +52,7 @@ function M.get_node_at_cursor()
   local line = cursor[1]
   if view.is_help_ui() then
     local help_lines = require'nvim-tree.renderer.help'.compute_lines()
-    local help_text = get_node_at_line(line+1)(help_lines)
+    local help_text = M.get_nodes_by_line(help_lines, 1)[line]
```

Comment:
This wasn't working properly when `hide_root_folder` set. Now it always starts at 1.